lib/upipe-blackmagic/sdi.c

```diff
@@ -0,0 +1,263 @@
+#include <stdbool.h>
```

Comment:
IIUC, there is nothing blackmagic-specific in this module, perhaps it should be moved to lib/upipe or upipe-modules ?

Reply:
True, although not sure what else could use it, maybe vanc filter?

Reply:
Another SDI output module (dektec or deltacast for instance).